report_xlsx/tests/test_report.py

```diff
@@ -55,3 +55,7 @@ def test_id_retrieval(self):
         # Typical call from render
         objs = self.xlsx_report._get_objs_for_report(self.docs.ids, {})
         self.assertEquals(objs, self.docs)
+
+    def test_currency_format(self):
+        usd = self.env.ref("base.USD")
+        self.assertTrue(usd.symbol in usd._get_format_amount())
```

Comment:
Better to test the expected string
```suggestion
        self.assertEqual(usd._get_format_amount(), '"$" #,##0,00')
```